3rdparty/lsqlite3/lsqlite3.c

```diff
@@ -1317,7 +1317,6 @@ static void db_update_hook_callback(void *user, int op, char const *dbname, char
     sdb *db = (sdb*)user;
     lua_State *L = db->L;
     int top = lua_gettop(L);
-    lua_Number n;
```

Comment:
I don’t want to diverge from upstream for something trivial like this.  I’ve already had to patch it to build cleanly as C++, I want to keep the patches minimal.